src/ffmpeg/ffmpeg_linux/ffmpeg_linux.c

```diff
@@ -1,4872 +1,6 @@
-/*
- * Copyright (c) 2000-2003 Fabrice Bellard
- *
- * This file is part of FFmpeg.
- *
- * FFmpeg is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- *
- * FFmpeg is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with FFmpeg; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
- */
+#include "libavutil/log.h"
```

Comment:
Shouldn't this file be entirely removed? I'm not sure what's left in it right now.

Reply:
What is defined in this file now is `mobileffmpeg_log_callback_function` which defines how the logs are handled for each platform.

Reply:
> how the logs are handled for each platform.

Ah so this is just the Linux-specific log handling? The file could possibly be renamed to reflect that?

Reply:
Yes exactly ! Yes you are right I will rename it